src/config.cpp

```diff
@@ -73,16 +97,16 @@ void parse_program_options(int argc, const char** argv)
              "attach to specific pid")
         ("quiet,q", po::bool_switch(&quiet),
              "suppress output")
-        ("verbose,v", po::bool_switch(&debug),
-             "verbose output")
-        ("extra-verbose,t", po::bool_switch(&trace),
-             "extra verbose output")
+        ("verbose,v", po::value(&verbosity)->zero_tokens(),
+             "verbose output (specify multiple times to get increasingly more verbose output)")
         ("mmap-pages,m", po::value(&config.mmap_pages)->default_value(16),
              "number of pages to be used by each internal buffer")
         ("readout-interval,i", po::value(&read_interval_ms)->default_value(100),
              "time interval between metric and sampling buffer readouts in milliseconds")
-        ("raw-tracepoint-event,e", po::value(&config.tracepoint_events),
+        ("tracepoint,t", po::value(&config.tracepoint_events),
              "enable global recording of a raw tracepoint event (usually requires root)")
+        ("event,e", po::value(&config.perf_events),
```

Comment:
Yes, this is a good moment to break with the options towards compatibility with `perf record`.